lib/environments/meteorEnvironment.js

```diff
@@ -308,6 +308,31 @@ export default {
     return moduleName;
   },
 
+  moduleSideEffectImports: ({ pathToCurrentFile, config }: Object): Array<string> => {
+    let basePath = '';
+    if (pathToCurrentFile.endsWith('.js')) {
+      basePath = pathToCurrentFile.slice(0, -3);
+    } else if (pathToCurrentFile.endsWith('.jsx')) {
+      basePath = pathToCurrentFile.slice(0, -4);
+    } else {
+      return [];
+    }
+    const moduleSpecifiers = [];
+    for (const ext of ['.html', '.css']) {
```

Comment:
This would be better as a `.forEach` loop instead:

```js
['.html', '.css'].forEach(ext => {
  ...
});
```

or maybe a map and filter.